packages/core/test/guards.test.ts

```diff
@@ -792,7 +744,7 @@ describe('not() guard', () => {
       {
         guards: {
           greaterThan10: ({ guard }) => {
-            return guard.params.value > 10;
+            return (guard as any).params.value > 10;
```

Comment:
Is strong typing possible?

Reply:
We’ll learn tomorrow :p but ye - it should be possible to have this one typed